schemas/test/en/test_calculated_summary.json

```diff
@@ -51,7 +51,11 @@
                                         "mandatory": true,
                                         "type": "Currency",
                                         "currency": "GBP",
-                                        "decimal_places": 2
+                                        "decimal_places": 2,
```

Comment:
Looks good 👍 I think it would be worth making this same change to `test_new_calculated_summary` so that they are still in sync, and then maybe have half your test cases use the new variant too

Reply:
Thanks Katie, with regards to adding the changes into `test_new_calculated_summary`, would I write my tests in the same file as I'm slightly unsure about the second part of the comment